oui/wired_laptop.sh

```diff
@@ -11,6 +11,7 @@ oui_wired_laptop=(
 	00:90:f5='CLEVO CO.|Clevo W110ER gaming laptop, Realtek RTL8111 Ethernet'
 	04:7d:7b='Quanta Computer Inc.|Lenovo ThinkPad X121e, Atheros AR8151 Ethernet'
 	10:bf:48='ASUSTEK COMPUTER INC.|Asus K45VM Notebook, Realtek PCIe GBE Family Controller'
+	28:d2:44='LCFC(HeFei) Electronics Technology Co., Ltd., Intel Corporation Ethernet Connection I218-LM'
```

Comment:
This has the wrong format. It should be in `aa:bb:cc='Organization name|Laptop & device name'` format.